x-pack/plugins/synthetics/common/constants/plugin.ts

```diff
@@ -10,13 +10,13 @@ import { i18n } from '@kbn/i18n';
 export const PLUGIN = {
   APP_ROOT_ID: 'react-uptime-root',
   DESCRIPTION: i18n.translate('xpack.uptime.pluginDescription', {
-    defaultMessage: 'Uptime monitoring',
+    defaultMessage: 'Synthetics monitoring',
```

Comment:
Where are these two values displayed? Are they displayed to the end user?

Reply:
i don't think so. 

Though the title goes here 
<img width="1578" alt="image" src="https://user-images.githubusercontent.com/3505601/163874412-c5e3ef76-85a7-4cb1-b1f5-da8b6f3f6d75.png">